### PR TITLE
chore(archive): renaming and refactor preparation

### PR DESCRIPTION
**Pull request by SionoiS**
Only renaming stuff for the next phase of refactor.

It's still kind of a mess but will improve soon.

### CODE REVIEW

**Review by gabrielmer — APPROVED** [review]
lgtmV2 😄

File: waku/waku_archive/driver/sqlite_driver/sqlite_driver.nim [hunk 1]
```diff
@@ -112,24 +112,24 @@ method getMessagesV2*(
 
 method getMessages*(
     s: SqliteDriver,
-    includeData = false,
-    contentTopic = newSeq[ContentTopic](0),
+    includeData = true,
```
Comment (Ivansete-status) [inline waku/waku_archive/driver/sqlite_driver/sqlite_driver.nim:115]: Why now set to `true`?

**Review by Ivansete-status — COMMENTED** [review]
I'm sorry but I don't feel confident about these changes.
Wouldn't it be easier to just separate the _StoreV2_ and _StoreV3_ logics in different files?

**Comment by SionoiS** [comment]
> I'm sorry but I don't feel confident about these changes. Wouldn't it be easier to just separate the _StoreV2_ and _StoreV3_ logics in different files?

You can't they share a common interface. All procs named `*V2` will be removed once Store v2 is removed.

I could create more `*_legacy` files for `index.nim` for egg.

**Comment by SionoiS** [comment]
Change of plan! I will open a feature branch and merge PRs and ask for review on that branch instead. It's too confusing otherwise.

**Pull request closed without merging**